File: iOSbeautyOfDevelop/chapter6/6.1.3/go2anotherView/go2anotherView/ViewController.m
//
//  ViewController.m
//  go2anotherView
//
//  Created by aa64mac on 8/25/15.
//  Copyright (c) 2015 cc. All rights reserved.
//

#import "ViewController.h"
#import "SceneBViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)buttonTouchUpInside:(id)sender {
    SceneBViewController *dest = [self.storyboard instantiateViewControllerWithIdentifier:@"SceneB"];
    dest.modalTransitionStyle = UIModalTransitionStyleFlipHorizontal;
    dest.modalPresentationStyle = UIModalPresentationFormSheet;
    [self presentViewController:dest animated:YES completion:nil];
}

@end

File: iOSbeautyOfDevelop/chapter6/6.1.3/go2anotherViewS/go2anotherViewS/ViewController.swift
//
//  ViewController.swift
//  go2anotherViewS
//
//  Created by aa64mac on 8/25/15.
//  Copyright (c) 2015 cc. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func buttonTouchUpInside(sender: AnyObject) {
        var dest = (self.storyboard!.instantiateViewControllerWithIdentifier("SceneB")) as! SceneBViewController
        dest.modalTransitionStyle = UIModalTransitionStyle.FlipHorizontal
        dest.modalPresentationStyle = UIModalPresentationStyle.FormSheet
        self.presentViewController(dest, animated: true, completion: nil)
    }

}

